src/serlo-editor/plugins/audio/editor.tsx

```diff
@@ -5,12 +5,15 @@ import type { AudioProps } from '.'
 import { parseAudioUrl, AudioRenderer } from './renderer'
 import { AudioToolbar } from './toolbar'
 import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
 
 export const AudioEditor = (props: AudioProps) => {
   const { focused, state } = props
   const [showSettingsModal, setShowSettingsModal] = useState(false)
   const [src, type] = parseAudioUrl(state.src.value)
   const couldBeValid = type !== undefined
+  // i18n string
+  const { audioUrl } = useEditorStrings().plugins.audio
```

Comment:
works, but I suggest to go with `const audioStrings = useEditorStrings().plugins.audio` and `audioStrings.audioUrl` to avoid confusions. Then you don't need a comment.